connectors/src/connectors/intercom/lib/utils.ts

```diff
@@ -149,7 +149,7 @@ export async function getParentIdsForArticle({
   connectorId: number;
   parentCollectionId: string;
   helpCenterId: string;
-}) {
+}): Promise<[string, string, ...string[], string]> {
```

Comment:
I assume articles can't be at the root of a help center (not inside a collection), correct?

Reply:
That's an interesting one, AFAICT it is possible to create an orphaned article in Intercom by deleting the collection but these are not displayed correctly in Intercom nor in our connector

Reply:
Here we don't fetch orphaned articles so the type is valid here